Guardfile

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# More info at https://github.com/guard/guard#readme
+
+# Limit directories to watch https://github.com/guard/guard/wiki/Guardfile-DSL---Configuring-Guard#directories
+directories(%w[lib spec]) \
```

Comment:
The select should apply to the %w(lib spec) first before assigning 'directories'. Using a continuation like this makes it a bit harder to spot, but I've run it though the dsl and it throws an error

Reply:
done, ta